File: JAVA/级数求和.java
//题目链接:"https://www.luogu.org/problemnew/show/P1035"
package com.company;
import java.util.*;
public class Main {

    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        double sum=0;
        int m=1;
        double n=s.nextDouble();
        while(true){
            if(n<sum)
                break;
            sum=1.0/m+sum;
            m++;
        }
        System.out.print(m-1);
    }
}

File: 洛谷/级数求和.cpp
//ÌâÄ¿Á´½Ó:"https://www.luogu.org/problemnew/show/P1035" 
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int n;
	cin>>n;
	double sum=0,a=1;
	while(1)
	{
		sum+=1/a;
		if(sum>n)
			break;
		a+=1;
	}
	cout<<a;
	return 0;
}
